packages/libs/wdk-client/src/Components/Display/CollapsibleSection.tsx

```diff
@@ -15,7 +15,7 @@ interface Props {
   isCollapsed?: boolean;
   onCollapsedChange: (isCollapsed: boolean) => void;
   headerContent: React.ReactNode;
-  headerComponent?: React.ReactType;
+  headerComponent?: React.FC | React.ComponentType;
```

Comment:
`ComponentType` is a union of `ComponentClass` and `FunctionComponent`.
```suggestion
  headerComponent?: React.ComponentType;
```